projects/cashmere/src/lib/picklist2/value-utils.ts

```diff
@@ -0,0 +1,40 @@
+const unescapedHTMLExp = /[&<>"']/g;
+const hasUnescapedHTMLExp = RegExp(unescapedHTMLExp.source);
+const htmlEscapes = {
+    '&': '&amp;',
+    '<': '&lt;',
+    '>': '&gt;',
+    '"': '&quot;',
+    '\'': '&#39;'
+};
+
+export function escapeHTML(string: string) {
+    return (string && hasUnescapedHTMLExp.test(string)) ?
+        string.replace(unescapedHTMLExp, chr => htmlEscapes[chr]) :
```

Comment:
@atkulp had a good idea when we were doing something like this in our code... Let the browser do the escaping for you. 

```
document.createElement('p'); // Throw-away element, or keep to reuse, but not in DOM
p.innerText = "<div>testing>br>&blah";  // Setting inner text causes escaping
p.innerHTML; // Get that escaped text back
```

Reply:
love it, I've implemented this